README.md

```
@@ -1,21 +1,172 @@
+Free typing game for programmers. You can import any GitHub repository and practice typing through real code.
 
-## 動作要件
-* ruby 3.x.x
-* rails 7.x.x
-* React x.x.x
+## URL
 
-## インストールと起動
-``` bash
-$ hoge
+https://typrogram.com
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)